sample-kotlin-dagger-2.1x/src/main/java/com/infullmobile/android/infullmvp/sample_dagger21x/application/SampleApp.kt

```diff
@@ -22,4 +27,8 @@ class SampleApp : Application(), HasActivityInjector {
 
     override fun activityInjector(): AndroidInjector<Activity> = dispatchingAndroidInjector
 
+    override fun fragmentInjector(): AndroidInjector<android.app.Fragment> = dispatchingFragmentInjector
```

Comment:
was is das ?

Reply:
You need it have Fragments injected in the app. Android's fragments or support ones. Technically you can have both, for whatever reason.

Forgot about import alias, so I may opt out for a `SupportFragment` alias instead of full package path.